package.json

```diff
@@ -43,6 +43,7 @@
     "babel-loader": "8.1.0"
   },
   "dependencies": {
+    "@babel/core": "^7.12.13",
```

Comment:
Is this really needed?

Reply:
I'm not sure, but it complained about unable to resolve babel/core in the first commit's lint and build steps: https://github.com/storybookjs/presets/runs/1855759502

Reply:
The build run perhaps explains it better:
```
Error: Cannot find module '@babel/core'
 babel-loader@8 requires Babel 7.x (the package '@babel/core'). If you'd like to use Babel 6.x ('babel-core'), you should install 'babel-loader@7'.
 ```
 https://github.com/storybookjs/presets/runs/1855759687

Reply:
👌🏻  I think it's because other presets are listing it as peerDep (like https://github.com/storybookjs/presets/blob/master/packages/preset-create-react-app/package.json#L40) but nobody provides it, we were in a magical state before but the bump in ant preset broke that. Nevermind, the dependencies are a mess in this repo for now so let's go with your option ;)